obal/data/module_utils/obal.py

```diff
@@ -4,21 +4,30 @@
 import subprocess
 
 
-def specfile_macro_lookup(specfile, macro_str):
+def specfile_macro_lookup(specfile, macro_str, scl=None, dist=None, macros=None):
     """expand a given macro from a specfile"""
-    cmd = [
+    command = [
         'rpmspec',
         '--query',
         '--queryformat',
         macro_str,
-        '--undefine',
-        'dist',
-        '--undefine',
-        'foremandist',
```

Comment:
Is this intentional? When you execute this on a machine which has `foreman-build` installed, this will get weird NEVRs.

Reply:
Yes, my intent was to have the projects define this if need be using the new ability to pass macros per tag since this was quite Foreman specific.

Reply:
ok